packages/odyssey-react-mui/src/labs/DataView/DataView.test.tsx

```diff
@@ -0,0 +1,89 @@
+/*!
+ * Copyright (c) 2024-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+import { DataView } from "./index";
+import {
+  data,
+  columns,
+} from "@okta/odyssey-storybook/src/components/odyssey-labs/DataView/personData";
+import { filterData } from "@okta/odyssey-storybook/src/components/odyssey-labs/DataView/dataFunctions";
+
+const getData = ({ ...props }) => {
+  return filterData({ data, ...props });
+};
+
+const testView = (
+  <DataView
+    availableLayouts={["table"]}
+    getData={getData}
+    tableLayoutOptions={{
+      columns: columns,
+    }}
+    hasSearch
+    hasPagination
+    paginationType="loadMore"
+    resultsPerPage={20}
+  />
+);
+
```

Comment:
I don't like doing this up at the top of the file. Even if redundant, I like my tests verbose, so I can easily see what's wrong, and I can easily add to that test.

I like having these repeated in each test because each one tests something different, and it's easier to see what the test is doing with what values when it's added to each test much like how we'd add this to a story in Storybook.